src/Plugin.Maui.Calendar/Shared/Controls/DayView.xaml

```diff
@@ -13,36 +13,73 @@
             <conv:StrokeShapeConverter x:Key="StrokeShapeConverter" />
         </ResourceDictionary>
     </ContentView.Resources>
-    <Border
+    <Grid
+        Padding="0"
+        Margin="0"
+        HeightRequest="{Binding DayViewSize, Mode=OneWay}"
+        WidthRequest="{Binding DayViewSize, Mode=OneWay}"
+        IsVisible="{Binding IsVisible, Mode=OneWay}">
+        <Border
         Padding="0"
         BackgroundColor="{Binding BackgroundColor, Mode=OneWay}"
         HeightRequest="{Binding DayViewSize, Mode=OneWay}"
         HorizontalOptions="Center"
-        IsVisible="{Binding IsVisible, Mode=OneWay}"
         Stroke="{Binding OutlineColor, Mode=OneWay}"
         StrokeShape="{Binding DayViewCornerRadius, Converter={StaticResource StrokeShapeConverter}, Mode=OneWay}"
         WidthRequest="{Binding DayViewSize, Mode=OneWay}">
+            <Border.GestureRecognizers>
+                <TapGestureRecognizer Tapped="OnTapped" />
+            </Border.GestureRecognizers>
+        </Border>
         <FlexLayout
             AlignItems="Center"
             Direction="{Binding EventLayoutDirection}"
             JustifyContent="Center">
+            <Grid HeightRequest="4"/>
             <Label
                 FontSize="{Binding FontSize, Mode=OneWay}"
                 HorizontalTextAlignment="Center"
                 Style="{Binding DaysLabelStyle, Mode=OneWay}"
                 Text="{Binding Date.Day, Mode=OneWay}"
                 TextColor="{Binding TextColor, Mode=OneWay}" />
-            <Border
-                Padding="0"
-                BackgroundColor="{Binding EventColor, Mode=OneWay}"
-                HeightRequest="8"
-                IsVisible="{Binding IsEventDotVisible, Mode=OneWay}"
-                StrokeShape="RoundRectangle 4"
-                WidthRequest="8" />
+            <StackLayout Orientation="Horizontal" HeightRequest="8">
```

Comment:
We can utilize HorizontalStackLayout for better performance here

Reply:
sure thing